src/utils/trigger-input-change-event.js

```diff
@@ -5,9 +5,18 @@ This source code is licensed under the MIT license found in the
 LICENSE file in the root directory of this source tree.
 */
 // @flow
-/* global window */
+/* global window document */
```

Comment:
Sorry I was not clear earlier. Both functions should be encapsulated in the input file since it's not used anywhere else

Reply:
how about moving the trigger input change event into the base-input but leaving `createEvent` as a util? it isn't used anywhere else but it is very general as a safe way to create event objects down to ie11. or we could polyfill `Event` during build and not worry about it